docs/reference/setup/install/docker.asciidoc

```diff
@@ -1,32 +1,52 @@
 [[docker]]
 === Install Elasticsearch with Docker
 
-Elasticsearch is also available as a Docker image.
-The image is built with {xpack-ref}/index.html[X-Pack] and uses https://hub.docker.com/_/centos/[centos:7] as the base image.
-The source code can be found on https://github.com/elastic/elasticsearch-docker/tree/{branch}[GitHub].
+Elasticsearch is also available as Docker images.
+The images use https://hub.docker.com/_/centos/[centos:7] as the base image and
+are available with {xpack-ref}/index.html[X-Pack].  The source code can be found
+on https://github.com/elastic/elasticsearch-docker/tree/{branch}[GitHub].
 
-==== Security note
+==== Image types
 
-NOTE: {xpack-ref}/index.html[X-Pack] is preinstalled in this image.
-Please take a few minutes to familiarize yourself with {xpack-ref}/security-getting-started.html[X-Pack Security] and how to change default passwords. The default password for the `elastic` user is `changeme`.
+The images are available in three different configurations or "flavours". The
```

Comment:
I couldn't find that in the guide. Do we have an official dialect, @hstephenson?

Reply:
Switched to US spelling since I want to ship this and it doesn't really hurt anything except my cultural pride (which is not particularly relevant, let's face it).